### DISCUSSION TITLE
default.metallib not found, loading from source

### DISCUSSION THREAD

**Discussion by zy445566** [withcatai/node-llama-cpp#82] [Q&A] [answered]
Find a problem can't solve someone can help me，thx

OS:  macOS Monterey 12.5.1
node version:  v16.15.1
ERROR:
llama_new_context_with_model: n_ctx      = 4096
llama_new_context_with_model: freq_base  = 10000.0
llama_new_context_with_model: freq_scale = 1
llama_new_context_with_model: kv self size  = 2048.00 MB
ggml_metal_init: allocating
ggml_metal_init: found device: Intel(R) Iris(TM) Plus Graphics
ggml_metal_init: picking default device: Intel(R) Iris(TM) Plus Graphics
ggml_metal_init: default.metallib not found, loading from source
zsh: segmentation fault  node index.mjs

**Comment by giladgd** [comment, marked as answer]
@zy445566 Since you're using a Mac with an Intel chip, you probably should [disable Metal support](https://withcatai.github.io/node-llama-cpp/guide/Metal)
